integrations/langfuse.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: integration
+name: langfuse
+description: langfuse-haystack integrates tracing capabilities into [Haystack](https://github.com/deepset-ai/haystack) (2.x) pipelines using [Langfuse](https://langfuse.com/). This package enhances the visibility of pipeline runs by capturing comprehensive details of the execution traces, including API calls, context data, prompts, and more. Whether you're monitoring model performance, pinpointing areas for improvement, or creating datasets for fine-tuning and testing from your pipeline executions, langfuse-haystack is the right tool for you.
```

Comment:
This text will be shown on the integration card and it's a bit too long to fit into the card. Can you make it shorter?
Example 👇 
![image](https://github.com/deepset-ai/haystack-integrations/assets/25897116/00ea4921-4c91-42e7-921f-2c9b5b67ffba)

We can use the longer description in the `Overview` section below